velox/docs/monthly-updates/november-2023.rst

```diff
@@ -0,0 +1,96 @@
+
+
+Core Library
+============
+* Add support to export constant vector as Arrow REE array. :pr:`7327`, :pr:`7398`
+* Add `libcpr` a lightweight http client. :pr:`7385`
+* Add spilling support for distinct aggregation. :pr:`7305`
+* Add DirectBufferedInput: a selective BufferedInput without caching. :pr:`7217`
+* Add config ``merge_exchange.max_buffer_size`` to limit the total memory used by exchange clients. :pr:`7410`
+* Add configs ``sort_writer_max_output_rows`` and ``sort_writer_max_output_bytes`` to limit memory usage of sort writer. :pr:`7339`
+* Add support for TIMESTAMP type in Arrow bridge. :pr:`7435`
+* Add termination time to TaskStats. :pr:`7479`
+* Add Presto Type Parser based on Flex and Bison. :pr:`7568`
+* Add PrestoQueryRunner that can be used to verify test results against Presto. :pr:`7628`
+* Add support for named row fields in type signature and binding. :pr:`7523`
+* Add support for spilling aggregations over sorted inputs. :pr:`7526`
+* Add support to shrink cache if clients such as Prestissimo detect high memory usage on a worker. :pr:`7547`, :pr:`7645`
+* Add support to lazily create the spill directory. :pr:`7660`
+* Add ``RowVector::childAt(name)`` API. :pr:`7737`
+* Add DuckDB version 0.8.1 as an external dependency and remove DuckDB amalgamation. :pr:`6725`
+* Fix distinct aggregations with global grouping sets to return a row per set with the group id on empty input. :pr:`7353`
+* Fix incorrect runtime stats reporting when memory arbitration is triggered. :pr:`7394`
+* Fix Arrow bridge to ensure the null_count is always set and add support for null constants. :pr:`7411`
+* Fix ``Timestamp::toMillis()`` to overflow only if the final result overflows. :pr:`7506`
+* Fix DECIMAL to VARCHAR cast by adding trailing zeros when the value is `0`. :pr:`7588`
+* Fix producing wrong result from aggregations over sorted inputs with spilling. :pr:`7764`
+* Upgrade Arrow dependency to 14.0.1 from 13.0.0.
+
+Presto Functions
+================
+* Add :func:`cosine_similarity` scalar function.
+* Add support for INTERVAL DAY TO SECOND type input to :func:`plus`, :func:`minus`, :func:`multiply` functions.
+* Add support for combination of TIMESTAMP, INTERVAL DAY TO SECOND type inputs to :func:`plus`, :func:`minus` functions.
+* Add support for INTERVAL DAY TO SECOND, DOUBLE input arguments to :func:`divide` function.
+* Add support to allow non-constant IN list in IN Presto predicate. :pr:`7497`
+* Fix :doc:`bitwise shift functions</functions/presto/bitwise>` to accept shift value `0`.
+* Fix :func:`entropy` aggregate function to return `0.0` on null inputs.
+* Fix :func:`array_sort` function from producing invalid dictionary vectors. :pr:`7800`
+* Fix :func:`lead`, :func:`lag` window functions to return null when the offset is null. :pr:`7254`
+* Fix :doc:`url_extract_*</functions/presto/url>` functions to return null on malformed inputs and support absolute URIs.
+* Register :func:`array_frequency` function for all primitive types.
```

Comment:
Move next to other scalar functions.

Overall, make sure to group related items together.